packages/bisect-summary/bisect-summary.0.3/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/lindig/bisect-summary/archive/0.2.zip"
```

Comment:
The URL now points to the 0.3 version. This was rooted in the (at least for me annoying) inability to have all opam-related records in the same repo when hosting on GitHub: you can't know the checksum of the release you are about to tag.